### ISSUE TITLE
Generating model file for a different number pattern

### ISSUE THREAD

**Issue by Raulie** [card-io/card.io-iOS-source#50]
I need to scan an ATM card that has a different pattern. 

I've made some quick changes to the source code, and replaced the Visa pattern found in n_vseg.cpp. Luckily, it's perfectly scanning the ATM:
![atm_card](https://cloud.githubusercontent.com/assets/16032065/11674706/b6aa4ada-9df9-11e5-99aa-1aedd105ec92.png)

The thing is that I also need to scan Visa cards. After trying to understand the code, I think that I need to modify modelm_befe75da.cpp in order to include the new pattern. I would like to replace the Amex pattern with the ATM one, but I have no idea of how to modify/generate this file.

Is there a guide that explains how to generate this model file?

**Comment by dgoldman-ebay** [comment]
Wow, good work! I'm actually surprised you could get as far as you already have.

Unfortunately, changing the deep-learning models is both difficult and undocumented. We have an entire toolkit of Python and other scripts that are needed for doing so, but no one has had the time to organize and document these.

This is more @josharian's area of expertise than mine. Perhaps he can offer some advice.

**Comment by S8ER01Z** [comment]
Raulie did you manage to get this done?  I am in need of something similar to what you posted.

**Comment by Raulie** [comment]
@S8ER01Z no, I couldn't. I don't even know if it's possible, since in addition from Theano, they use an in-house Python library in order to build the models.
